util/Migrator/DbScripts/2024-08-13_00_UpdateCollectionManagementSettingsDbDefaults.sql

```diff
@@ -0,0 +1,312 @@
+-- Objective: 'LimitCollectionCreationDeletion' and 'AllowAdminAccessToAllCollectionItems' columns were set to 1 by
+-- default as a transitional measure. This script updates the default values to 0 to mirror the values set in code.
+
+-- Step 1: Drop existing default constraints
+ALTER TABLE [dbo].[Organization] DROP CONSTRAINT [DF_Organization_LimitCollectionCreationDeletion];
+GO
+ALTER TABLE [dbo].[Organization] DROP CONSTRAINT [DF_Organization_AllowAdminAccessToAllCollectionItems];
+GO
```

Comment:
We should check if the constraint exists before trying to drop it: [example from our code](https://github.com/bitwarden/server/blob/main/util/Migrator/DbScripts/2024-06-11_00_FixProviderInvoiceItem.sql#L9-L15), [related SO answer](https://stackoverflow.com/a/13186918). Although that doesn't discriminate between tables.

@rkac-bw what's your recommendation?

Reply:
It's much cleaner to check if exists